src/ripple/crypto/impl/ECIES.cpp

```diff
@@ -97,6 +99,11 @@ static void getECIESSecret (const openssl::ec_key& secretKey, const openssl::ec_
     memset (hbuf, 0, ECIES_KEY_LENGTH);
 }
 
+static void getECIESSecret (uint256 const& secretKey, Blob const& publicKey, ECIES_ENC_KEY_TYPE& enc_key, ECIES_HMAC_KEY_TYPE& hmac_key)
+{
```

Comment:
Not specific to this code, but is there a reason rippled uses static functions instead on anon namespaces?